### PR TITLE
refactor(runtime): replace trace with println

### PR DESCRIPTION
**Pull request by orhun**
Fixes the following build failure:

```
error[E0432]: unresolved import `tracing`
  --> runtime/src/next/mod.rs:26:5
   |
26 | use tracing::{error, trace, warn};
   |     ^^^^^^^ help: a similar path exists: `shuttle_common::tracing`

```

### CODE REVIEW

File: runtime/Cargo.toml [hunk 1]
```diff
@@ -21,6 +21,7 @@ serde = { workspace = true }
 serde_json = { workspace = true }
 strfmt = { workspace = true }
 thiserror = { workspace = true }
+tracing = { workspace = true }
```
Comment (chesedo) [inline runtime/Cargo.toml:24]: Since no tracing is setup in the runtimes, we should replace those log lines with `println`s instead

**Review by chesedo — COMMENTED** [review]
We should use `println` instead

File: runtime/Cargo.toml (hunk 1 above)
Comment (orhun) [reply runtime/Cargo.toml]: Done: 8647dd3a352747ce042cc75b0e9ac9a44e879364

**Review by chesedo — APPROVED** [review]
LGTM

File: runtime/src/next/mod.rs [hunk 2]
```diff
@@ -69,7 +68,7 @@ impl Runtime for AxumWasm {
         request: tonic::Request<LoadRequest>,
     ) -> Result<tonic::Response<LoadResponse>, Status> {
         let wasm_path = request.into_inner().path;
-        trace!(wasm_path, "loading shuttle-next project");
+        println!("loading shuttle-next project");
```
[the remark this answers is not in the document]
Comment (orhun) [reply runtime/src/next/mod.rs]: 4a9eae582a0b4f4190f2e0448e97b524df929593

**Pull request merged**